triage-party/base/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: triage-party
+spec:
+  selector:
+    matchLabels:
+      app: triage-party
+  template:
+    metadata:
+      labels:
+        app: triage-party
+    spec:
+      containers:
+        - name: triage-party
+          image: triage-party:latest
```

Comment:
@codificat any reason we are using an imagestream here? I see that the imagestream is pointing to a fixed image in quay (`quay.io/thoth-station/triage-party:v1.4.0`) anyway. Would it be better to just use that and eliminate the ImageStream?